### PR TITLE
slides/tutorial-summit-networking: add slides

### PR DESCRIPTION
**Pull request by fjmolinas**
This is just a summarized version of what was already done for the Inria Academy. The introduction is shortened, but besides that its the same.

### CODE REVIEW

File: slides/tutorial-summit-networking/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Microcontroller based IoT: a hand-on course with RIOT</title>
```
Comment (aabadie) [inline slides/tutorial-summit-networking/index.html]: The title should be adapted to the tutorial name: "RIOT Summit - Networking Tutorial" or something like this

**Comment by aabadie** [comment]
And there's a typo in the commit message: `slides/tutorial-summit-netwowking: add slides` => `slides/tutorial-summit-networking: add slides`

File: slides/tutorial-summit-networking/index.html (hunk 1 above)
Comment (aabadie) [reply slides/tutorial-summit-networking/index.html]: Other summit tutorials have the title: `RIOT Summit - Beginner Tutorial` but maybe they should be adapted as well.

Comment (fjmolinas) [reply slides/tutorial-summit-networking/index.html]: I'll keep `RIOT Summit - Beginner Tutorial`, not sure how far we will get

**Review by emmanuelsearch — APPROVED** [review]
LGTM

**Pull request merged**